mst/page-index/page-index.js

```diff
@@ -58,6 +58,14 @@ module.exports = [
         title: "Manage-backups",
         href: "manage-backups"
       },
+      {
+        title: "Backups",
+        href: "backup-types"
```

Comment:
Was there supposed to be a page added for this in this PR?

Reply:
> Was there supposed to be a page added for this in this PR?

this was because of the merge conflict that had to be resolved.